sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -720,9 +720,27 @@ gnrc_netif_t *gnrc_netif_get_by_prefix(const ipv6_addr_t *prefix)
 int gnrc_netif_ipv6_group_join_internal(gnrc_netif_t *netif,
                                         const ipv6_addr_t *addr)
 {
-    unsigned idx = UINT_MAX;
+    uint8_t group_data[GNRC_NETIF_L2ADDR_MAXLEN];
+    unsigned idx;
 
+    /* can be called out of lock */
+    idx = gnrc_netif_ipv6_group_to_l2_group(netif, addr, group_data);
```

Comment:
Why `idx` - shouldn't that be `len`?

Reply:
I can rename it `res`, so I can reuse the variable, if that is preferred.

Reply:
I think the generated code would not change even if you introduced a new variable